### ISSUE TITLE
[Question] ToolBarItems in Shell

### ISSUE THREAD

**Issue by AnthonyLatty** [xamarin/Xamarin.Forms#6453]
### Description
I am using XF 4.0 with the Shell features. This is more of a question, i would like to know if Shell can add ToolBarItems similar to MainPage 

### Steps to Reproduce

Add below code in AppShell.xaml
```
 <Shell.ToolbarItems>
        <ToolbarItem Clicked="Account_Clicked"
                     IconImageSource="ic_account.png"
                     Order="Primary"
                     Priority="0"
                     Text="Account" />
        <ToolbarItem Clicked="Settings_Clicked"
                     Order="Primary"
                     Priority="1"
                     Text="Settings">
            <ToolbarItem.IconImageSource>
                <OnPlatform x:TypeArguments="ImageSource">
                    <On Platform="iOS" />
                    <On Platform="Android">ic_setting.png</On>
                </OnPlatform>
            </ToolbarItem.IconImageSource>
        </ToolbarItem>
    </Shell.ToolbarItems>
```

### Expected Behavior
ToolbarItems should be displayed.

### Actual Behavior
No ToolBarItems is shown

**Comment by samhouts** [comment]
@PureWeen Any thoughts?

**Comment by PureWeen** [comment]
@AnthonyLatty Yea! I feel like this is a good idea. Being able to just specify some persistent Menu items would be really useful. Like putting the about or other icons up there

**Comment by AnthonyLatty** [comment]
@PureWeen exactly...this functionality was vital in XF 3.6.xx. Now in XF 4.0 its missing. There are cases where you want to display toolbar items across pages. In XF 3.6.x.x, defining toolbar items in MainPage achieves this.

**Comment by PureWeen** [comment]
@AnthonyLatty I didn't realize that this worked in 3.6

So you're saying it broke in 4.0? Alright I'll prioritize this a bit higher than

**Comment by AnthonyLatty** [comment]
@PureWeen Yeah it definitely is broken for 4.0. The only difference in the code above is for traditional XF apps with MainPage it would be `MainPage.ToolBarItems`.

With shell automatically i thought it would be `Shell.ToolBarItems` but thats not the case. I will have to adjust my designs etc.

**Comment by Saikumar3002** [comment]
How can we add the Toolbar Items On Shell template in Xamarin. Forms

**Comment by cfb11** [comment]
> How can we add the Toolbar Items On Shell template in Xamarin. Forms

Today I don't have solutions for that question, anyone already has?

**Comment by AnthonyLatty** [comment]
What is the status on this @samhouts ?

**Comment by samhouts** [comment]
This is on the backlog. I don't currently have an estimate of when we may begin work on this item. Thank you for your patience!

**Comment by eramrit78** [comment]
The same issue come with me. We are try to add toolbar items on "Tabbed Pages" in shell but it not work . please make sure it work with "Content Page" as well as with "Tabbed Page"

Same issue #7351 
Thanks

**Comment by eramrit78** [comment]
!hope done in 5.0.1 release

**Comment by Rob97** [comment]
Please fix this ASAP especially in the 5.0.1 release! I'm in a really shitty delima where I call the shell navigation from ViewModel so the title and blue bar at the top don't show completely screwing up the same UI that works on other screens. This needs to be fixed ASAP.

**Comment by aodegov** [comment]
Hi there!

any news about global shell ToolbarItems: 
<Shell.ToolbarItems> 
                .............
</Shell.ToolbarItems>
, which can be visible on each page without copying the same items on each page?

**Comment by MuhammadJazab** [comment]
Hi there!

any update about shell ToolbarItems:
